settings.gradle

```diff
@@ -1 +1,2 @@
 include "marklogic-unit-test-modules", "marklogic-unit-test-client", "marklogic-junit5"
```

Comment:
Put these on separate lines - it makes life easier for Sameera later on when publishing each one at a time.